scripts/asm-installer/install_asm

```diff
@@ -2206,11 +2226,20 @@ EOF
   fi
 }
 
-ensure_istio_namespace_exists() {
+create_istio_namespace() {
+  info "Creating istio-system namespace..."
+  retry 2 kubectl create ns istio-system
```

Comment:
I think we should probably also check to see if it exists here and just return if it does.

Reply:
Done